lib/utils.ts

```diff
@@ -34,11 +34,11 @@ interface MinMaxInterface {
 export const getArrayMaxOrMinAfterIndex = (
```

Comment:
```suggestion
export const getTrendAfterIndex = (
```
or something like that, just need to change this name accordingly